Grammar/python.gram

```diff
@@ -1362,3 +1362,7 @@ invalid_replacement_field:
     | '{' a=':' { RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "f-string: expression required before ':'") }
     | '{' a='!' { RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "f-string: expression required before '!'") }
     | '{' a='}' { RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "f-string: empty expression not allowed") }
+    | '{' (yield_expr | star_expressions) "="? invalid_conversion_character
+invalid_conversion_character:
+    | a="!" &(':'|'}') { RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "f-string: missed conversion character") }
```

Comment:
maybe this is slightly better?


```suggestion
    | a="!" &(':'|'}') { RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "f-string: missing conversion character") }
```

Reply:
I will fix this in another push. Thanks for the suggestion!